src/types/common.d.ts

```diff
@@ -0,0 +1,22 @@
+export interface MusicAccidental {
+  note: number;
+  accidental: number;
+}
+
+export interface MusicNoteParts {
+  root: string;
+  accidental: string;
+  type?: string;
+}
+
+/** Range 0 - 11 */
```

Comment:
Please remove the comments, you could just do: `export type KeyValue = 0|1|2|3|4|5|6|7|8|9|10|11`

Reply:
I find this proposal much better and will show problems if any at compile time. Not at run time though.

Reply:
I go for it